GVRf/Framework/framework/src/main/java/org/gearvrf/GVRViewManager.java

```diff
@@ -694,14 +695,22 @@ protected void captureCenterEye() {
         if (mScreenshotCenterCallback == null) {
             return;
         }
+        GVRRenderTexture eyeCaptureRenderTexture = mRenderBundle.getEyeCaptureRenderTexture();
```

Comment:
check for multiview is in getEyeCaptureRenderTexture() which returns null if its not multiview.